tests/conftest.py

```diff
@@ -85,6 +86,14 @@ def cte_optimization_enabled(pytestconfig):
     return pytestconfig.getoption("enable_cte_optimization")
 
 
+@pytest.fixture(scope="session", autouse=True)
+def multithreading_mode(pytestconfig):
+    enabled = pytestconfig.getoption("multithreading_mode")
+    if enabled:
+        os.environ["SNOWPARK_MULTITHREADING_MODE"] = "true"
+    return enabled
+
+
```

Comment:
remember to run in stored proc, I remember previously when I called similar function in local testing test setup and it broke sproc test